email_mailbox/src/utils/electronEventInterface.js

```diff
@@ -417,20 +418,22 @@ const handleNewMessageEvent = async ({ rowid, params }) => {
         emailThreadId = emailWithMessageId.threadId;
       }
     }
-
+    const secure =
+      guestEncryption === 1 || guestEncryption === 3 ? true : false;
```

Comment:
You don't need a ternary operator if you need to set `true` or `false`
```
const secure = guestEncryption === 1 || guestEncryption === 3;
```
should be enough